contributing_to_docs/doc_guidelines.adoc

```diff
@@ -928,6 +924,68 @@ This renders as:
 +
 NOTE: Avoid using full command syntax inline with sentences.
 
+* When referring to a path to a location that the user has selected or created, treat the part of the path that the user chose as a replaceable value. For example:
++
+....
+Create a secret that contains the certificate and key in the `openshift-config` namespace:
+
+[source,terminal]
+----
+$ oc create secret tls <certificate> --cert=<path_to_certificate>/cert.crt --key=<path_to_key>/cert.key -n openshift-config
+----
+....
++
+This renders as:
++
+--
+> Create a secret that contains the certificate and key in the `openshift-config` namespace:
+>
+> ----
+> $ oc create secret tls <certificate> --cert=<path_to_certificate>/cert.crt --key=<path_to_key>/cert.key -n openshift-config
+> ----
+--
++
+The following example shows a more complex use of user-chosen elements and prescriptive placement:
++
+....
+Configure the disk encryption set under the `providerSpec` field in your machine set YAML file. For example:
++
+[source,yaml]
+----
+...
+providerSpec:
+  value:
+    ...
+    osDisk:
+      diskSizeGB: 128
+      managedDisk:
+        diskEncryptionSet:
+          id: /subscriptions/<subscription_id>/resourceGroups/<resource_group_name>/providers/Microsoft.Compute/diskEncryptionSets/<disk_encryption_set_name>
```

Comment:
For this second example, could I just show the path and not all this context?

Reply:
I think you can and should show only the path (l963) and I also don't think the "renders as" example that follows is necessary, either, unless I'm missing something.